apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -590,7 +653,12 @@ def parse(element_html, data):
             )
             answer["tag"] = search["tag"] if search is not None else None
 
-    if pl.get_string_attrib(element, "grading-method", "ordered") == "external":
+    if (
+        pl.get_enum_attrib(
```

Comment:
Why use this getter again? It looks like this attribute was retrieved above.

Reply:
nice catch. fixed.